test/libsolidity/syntaxTests/license/license_missing.sol

```diff
@@ -1 +1,2 @@
+// This test is actually useless, as the test suite adds the automatic preamble.
```

Comment:
Since you added metadata tests for this, it should be removed, right?

Reply:
The reason I kept is I hope the metadata test is temporary.

Reply:
OK. I'd put that it the comment (perhaps even referencing the metadata test and this file from metadata test) but now I understand the intention.